<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="PaintWidget" name="disWidget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnOpenFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>920</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnSaveFile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>920</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="TreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>960</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>960</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>880</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cBox0">
       <property name="text">
        <string>Layer0</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cBox1">
       <property name="text">
        <string>Layer1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cBox2">
       <property name="text">
        <string>Layer2</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header location="global">paintwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">treewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
